Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -428,23 +430,23 @@ extension DefaultAudioDevice {
                 isPlayerInterrupted = true
                 let _ = stopRendering()
             }
-        case AVAudioSessionInterruptionType.ended.rawValue:
+        case AVAudioSession.InterruptionType.ended.rawValue:
             configureAudioSessionWithDesiredAudioRoute(desiredAudioRoute: DefaultAudioDevice.kAudioDeviceBluetooth)
             restartAudioAfterInterruption()
         default:
             break
         }
     }
     
-    func onRouteChangeEvent(notification: Notification) {
+    @objc func onRouteChangeEvent(notification: Notification) {
         safetyQueue.async {
             self.handleRouteChangeEvent(notification: notification)
         }
     }
     
-    func appDidBecomeActive(notification: Notification) {
+    @objc func appDidBecomeActive(notification: Notification) {
         safetyQueue.async {
-            self.handleInterruptionEvent(type: Int(AVAudioSessionInterruptionType.ended.rawValue))
+            self.handleInterruptionEvent(type: Int(AVAudioSession.InterruptionType.ended.rawValue))
```

Comment:
I think the code will need a bit of refactoring after this PR - not sure why type is Int rather than AVAudioSession.InterruptionType.